tests/ProxyManagerTest/Functional/LazyLoadingGhostFunctionalTest.php

```diff
@@ -829,4 +830,37 @@ public function getPropertyAccessProxies()
             ],
         ];
     }
+
+    /**
+     * @dataProvider skipPropertiesFixture
+     *
+     * @param string   $className
+     * @param string   $propertyName
+     * @param string[] $properties
+     */
+    public function testSkipProperties($className, $propertyName, $properties)
```

Comment:
As discussed, this test should be duplicated into the `LazyLoadingGhostFunctionalTest`. This particular test should probably just mock out logic to verify that `$properties` is passed down to the generator and to the various hasher/signing logic stuff